pyrk/inp/validation.py

```diff
@@ -23,6 +23,25 @@ def validate_ge(valname, val, llim):
     else:
         return val
 
+def validate_g(valname, val, llim):
+    """Raises errors if the value is less than the lower limit (llim) or if it
+    is of the wrong type
+
+    :param valname: the name of the value being validated
+    :type valname: string
+    :param val: the value to be validated
+    :type val: should be a number (or pint.Quantity)
+    :param llim: the lower limit of acceptable value for val
+    :type llim: the same type as val
+    """
+    if not validate_num(valname, val) > llim:
+        msg = valname + " must be greater than"
+        msg += str(llim) + ".\n"
+        msg += "The value provided was : "
+        msg += str(val)
+        raise ValueError(msg)
+    else:
+        return val
```

Comment:
(needs tests though) please add tests for validate_g to the set of tests that test the validation functions.